src/components/Cards/PlayerCards.test.js

```diff
@@ -41,3 +43,50 @@ describe('on given a set of cards', () => {
     })
 })
 
+const helper = {
+    fetchCards () {
+        return axios.get('/api/cards')
+    }
+}
+
+describe('moxios', () => {
+    it('should install', () => {
+      let defaultAdapter = axios.defaults.adapter
+      moxios.install()
+      expect(axios.defaults.adapter === defaultAdapter).toEqual(false);
+      moxios.uninstall()
+    })
+
+    it('should uninstall', () => {
```

Comment:
What are these testing?

Reply:
If moxios set up correctly I think... I'm not really sure, I was just following a tutorial :D

Reply:
That's generally not the kind of thing you'd want to test, since it's not code that you've written.